src/actions.ml

```diff
@@ -257,6 +257,43 @@ let send_doc_url ~bot_info ~github_repo_full_name job_info =
   | _ ->
       Lwt.return_unit
 
+let update_bench_status ~bot_info job_info (gh_owner, gh_repo) ~external_id =
+  GitHub_queries.get_repository_id ~bot_info ~owner:gh_owner ~repo:gh_repo
+  >>= function
+  | Ok repo_id -> (
+      Lwt_io.print "Pushing status check for bench job.\n"
+      <&>
+      let url = f "https://gitlab.com/coq/coq/-/jobs/%d" job_info.build_id in
+      let state = job_info.build_status in
+      let context = "bench" in
+      ( match state with
+      | "success" ->
+          Lwt.return_ok (COMPLETED, Some SUCCESS, "Bench completed successfully")
+      | "created" ->
+          Lwt.return_ok
+            ( QUEUED
+            , None
```

Comment:
This is why nothing is shown when the checks are run. Should we change this to `Some NEUTRAL`?

Reply:
By the way, `NEUTRAL` will get completely ignored if we also don't change `QUEUED` to `COMPELTED` IIUC. This is what I have done. That's a good point about neutral however.

Reply:
I tried action_requires but that fails the check which isn't great. I've switched back to neutral.

Reply:
Which by the way is all happening in the other PR not this one. I think we can merge this one now @Zimmi48.

Reply:
Unless we want to make the neutral happen here...